modules/app-configuration/configuration-store/README.md

```diff
@@ -862,26 +862,16 @@ Optional. A list of IP configurations of the private endpoint. This will be used
 
 | Name | Required | Type | Description |
 | :-- | :-- | :--| :-- |
-| [`groupId`](#parameter-privateendpointsipconfigurationsgroupid) | Yes | string |  |
-| [`memberName`](#parameter-privateendpointsipconfigurationsmembername) | Yes | string |  |
 | [`name`](#parameter-privateendpointsipconfigurationsname) | Yes | string |  |
-| [`privateIpAddress`](#parameter-privateendpointsipconfigurationsprivateipaddress) | Yes | string |  |
-
-### Parameter: `privateEndpoints.ipConfigurations.groupId`
-- Required: Yes
-- Type: string
-
-### Parameter: `privateEndpoints.ipConfigurations.memberName`
-- Required: Yes
-- Type: string
```

Comment:
@AlexanderSehr Q/ Given we're not flattening the properties anymore. These properties are no longer surfaceable in the readme. Thoughts?

Reply:
Hey @ahmadabdalla, I believe the reason is that instead of definiting a dedicated UDT that you reference, it is a nested one with a lack of metadata at that. I'd argue this could be an additional feature request and not part of this PR

Reply:
No problems. Was a clarification. Thanks @AlexanderSehr